pkg/storage/unified/apistore/store.go

```diff
@@ -104,6 +110,14 @@ func NewStorage(
 		opts: opts,
 	}
 
+	if opts.RequireDeprecatedInternalID {
+		node, err := snowflake.NewNode(rand.Int63n(1024))
+		if err != nil {
+			return nil, nil, err
+		}
+		s.snowflake = node
```

Comment:
where is this used?

Reply:
https://github.com/grafana/grafana/pull/98311/files#diff-a6d40da7a17d43622287c2147606ef055a394f2110ac283dea3c10d5fe64f216R64

Reply:
It is used to set a fake "internal id" when one does not exist.  We need something to keep the legacy:
```
/api/dashbaord/id/{id}
```
working even if there is no sequential ID from the database.

Reply:
thanks, i missed it in the other file